components/web/apps/org.wso2.carbon.apimgt.editor.app/src/main/fragments/action-menu-api-details/action-menu-api-details.hbs

```diff
@@ -19,10 +20,11 @@
             </a>
         </li>
         <li>
-            <a href="{{env.contextPath}}/version?name=cha123&amp;version=1.0.0&amp;provider=admin" title="Create New Version">
+        {{!--{{env.contextPath}}/version?name=cha123&amp;version=1.0.0&amp;provider=admin--}}
+            <a href="#" onclick="return alert('This function is not supported yet')" title="Delete">
```

Comment:
Adding default js alert isn't recommend to use in wso2 products. use standard lib to show notifications and errors